src/Arcus.Templates.Tests.Integration/Health/HealthEndpointService.cs

```diff
@@ -39,11 +49,23 @@ public HealthEndpointService(TestConfig configuration, ITestOutputHelper outputW
         /// </summary>
         public async Task<HttpResponseMessage> GetAsync()
         {
-            _outputWriter.WriteLine("GET {0} ->", _healthEndpoint);
-            var response = await HttpClient.GetAsync(_healthEndpoint);
-            _outputWriter.WriteLine("<- {0}", response.StatusCode);
+            return await GetAsync(request => { });
```

Comment:
What is this request for? Can't we make it optional?

Reply:
If we want to manipulate the request before sending. Like in the shared access auth test to add headers.

This overlaod doesn't add anything so we have to pass along an empty action.